Atcoder_#357/b_uppercase_and_lowercase.rb

```diff
@@ -0,0 +1,14 @@
+# 入力を取得
+S = gets.chomp
+
+# 大文字と小文字のカウント
+uppercase_count = S.count("A-Z")
+lowercase_count = S.count("a-z")
```

Comment:
```suggestion
uppercase_count = S.count('A-Z')
lowercase_count = S.count('a-z')
```
文字列が固定されている場合は、
基本的にシングルクウォートにしましょう。

Reply:
ありがとうございます！
こちら以下のコミットで修正いたしました！
ac7aa9afaa5dae5daa717d29f98da00df784bd91